FlowCryptCommon/UIVIewExtensions.swift

```diff
@@ -0,0 +1,107 @@
+//
+// © 2017-2019 FlowCrypt Limited. All rights reserved.
+//
+
+import UIKit
+
+public extension UIView {
+    @discardableResult
+    func bordered(color: UIColor, width: CGFloat) -> Self {
```

Comment:
I like these kind of extensions, they increase code clarity, remove repetitive code, improve readability. Well done.